src/polar/Pie.tsx

```diff
@@ -58,12 +59,19 @@ type PieLabelLine =
   | ((props: any) => ReactElement<SVGElement>)
   | SVGProps<SVGPathElement>
   | boolean;
-export type PieLabel<P = any> =
+
+export type PieLabelProps = PieSectorData &
+  GeometrySector & {
```

Comment:
interesting... This may get passed but I wonder if we actually want it on the type here. Adding something to the type is saying "this belongs here" to consumers, the geometry sector feels out of place. As does tooltipPayload...

Hmm - thoughts @PavelVanecek ?

Reply:
Agree that this looks out of place but if it's there then it's part of the external API. If that was intentional or accidental - we might never know.

If it's passed now, and it's passed as `any` then that's twice as bad. Adding types is an improvement - and it allows us to later mark it as obsolete/deprecated and start working towards removing it. If we decide that's the way to go of course.

Reply:
We would only make our removal of any more difficult, if we also at the same time would try to type things in meaningful ways 😓 
For now I think merely having types that do not include any would be a great step forward in reliability.